core/src/main/java/gatt/CoroutinesGatt.kt

```diff
@@ -44,6 +44,10 @@ class CoroutinesGatt internal constructor(
     override val services: List<BluetoothGattService> get() = bluetoothGatt.services
     override fun getService(uuid: UUID): BluetoothGattService? = bluetoothGatt.getService(uuid)
 
+    override fun requestConnectionPriority(
```

Comment:
How do we treat them differently internally? Is this a bluetooth concept or just one within our stack?

Reply:
> How do we treat them differently internally?

What are you referring to when you say "them"?

> Is this a bluetooth concept or just one within our stack?

It is a BLE concept. Has to do with how active the radios are for communication. Android sends a request to the remote peripheral to adjust its communication parameters.